docs/connectors/sources/http.md

```diff
@@ -1,16 +1,87 @@
-# HTTP Input
+# HTTP input connector
 
-Feldera supports pushing data to a SQL table over HTTP.  For more information,
-see
+Feldera supports directly pushing data to a SQL table over HTTP.
 
-* [Tutorial section](/docs/tutorials/basics/part2) on HTTP-based input and
-  output.
+* This input connector is ephemeral: it is created when the HTTP
+  connection is opened, and is deleted once it is closed.
 
-* [REST API documentation](https://www.feldera.com/api/push-data-to-a-sql-table)
-  for the `/ingress` endpoint.
+* It is the only input connector not created and managed by the user.
 
-:::caution Under Construction
+* Usage is through a special endpoint:
 
-This section is under construction.
+  **[/v0/pipelines/:pipeline_name/ingress/:table_name?format=...](https://www.feldera.com/api/push-data-to-a-sql-table)**
 
-:::
+* Data input format is specified through URL query parameters
+  (e.g., `format=...`, and more depending on format).
+
+## Example usage
+
+We will insert for pipeline `supply-chain-pipeline` into table `product`.
+
+### curl
+
+#### One row
+```bash
+curl -i -X 'POST' \
+  http://localhost:8080/v0/pipelines/supply-chain-pipeline/ingress/product?format=json \
+  -d '{"insert": {"pid": 0, "name": "hammer", "price": 5.0}}'
+```
+
+#### One row while providing authorization header
+```bash
+curl -i -H "Authorization: Bearer <API-KEY>" -X 'POST' \
+  http://localhost:8080/v0/pipelines/supply-chain-pipeline/ingress/product?format=json \
+  -d '{"insert": {"pid": 0, "name": "hammer", "price": 5.0}}'
+```
+
+#### Multiple rows (note: URL parameter `array=true`)
```

Comment:
Maybe worth having two sections: "Multiple rows" as newline-delimited JSON, and "Multiple rows as a JSON array"